zipkin-cassandra/src/schema/cassandra-schema-cql3.txt

```diff
@@ -0,0 +1,73 @@
+CREATE KEYSPACE "Zipkin" WITH replication = {'class': 'NetworkTopologyStrategy', 'us-west': '1'}  AND durable_writes = true;
```

Comment:
If a single node fails, 1/N of your data will disappear. If that's ok, then leaving this at 1 is fine.